### PR TITLE
Update mockito to v2.10

### PR DESCRIPTION
**Pull request by benoitf**
requires
- [ ] https://github.com/eclipse/che-dependencies/pull/62
- [ ] https://github.com/eclipse/che/pull/6394

### What does this PR do?
Update mockito to v2.10
it allows to start to use Java9 as runtime

#### Release Notes
N/A

#### Docs PR
N/A

### CODE REVIEW

**Comment by benoitf** [comment]
ci-build

File: plugins/plugin-hosted/codenvy-machine-hosted/src/test/java/com/codenvy/machine/backup/WorkspaceFsBackupSchedulerTest.java [hunk 1]
```diff
@@ -171,7 +172,7 @@ public void shouldNotBackupWSWithNonRunningStatus() throws Exception {
     // add this verification with timeout to ensure that thread executor had enough time before verification of call
     verify(backupManager, timeout(2000)).backupWorkspace(WORKSPACE_ID_1);
     verify(backupManager, timeout(2000)).backupWorkspace(WORKSPACE_ID_2);
-    verify(backupManager, timeout(2000).never()).backupWorkspace("ws3");
```
Comment (garagatyi) [inline plugins/plugin-hosted/codenvy-machine-hosted/src/test/java/com/codenvy/machine/backup/WorkspaceFsBackupSchedulerTest.java]: @benoitf Can you elaborate on why did you change it?

Comment (benoitf) [reply plugins/plugin-hosted/codenvy-machine-hosted/src/test/java/com/codenvy/machine/backup/WorkspaceFsBackupSchedulerTest.java]: sure, 
>
https://static.javadoc.io/org.mockito/mockito-core/1.10.19/org/mockito/verification/VerificationWithTimeout.html
 never()
Deprecated. 
Validation with timeout combined with never simply does not make sense, as never() will typically immediately pass, and therefore not wait the timeout. The behaviour you may be looking for is actually provided by after().never().
To avoid compilation errors upon upgrade the method is deprecated and it throws a "friendly reminder" exception.
In a future release we will remove timeout(x).atMost(y) and timeout(x).never() from the API.
Do you want to find out more? See issue 235

In 2.x the deprecated stuff has been removed so I switch to after(timeout).never()

**Comment by riuvshin** [comment]
ci-test

**Comment by riuvshin** [comment]
ci-build

File: plugins/plugin-hosted/codenvy-machine-hosted/src/test/java/com/codenvy/machine/backup/WorkspaceFsBackupSchedulerTest.java (hunk 1 above)
Comment (garagatyi) [reply plugins/plugin-hosted/codenvy-machine-hosted/src/test/java/com/codenvy/machine/backup/WorkspaceFsBackupSchedulerTest.java]: tnx for the explanation

**Comment by benoitf** [comment]
ci-build
